### DISCUSSION TITLE
How to add a new line in the task help?

### DISCUSSION THREAD

**Discussion by joaonc** [pyinvoke/invoke#1004] [Q&A]
Trying this
```python
@task(
    help={
        'my_option': 'Long description.\nSecond line.'
    },
)
def my_task(c, my_option: str):
    ...
```
But the `\n` is not working with `inv --help my_task`.
Running on Windows 11, python 3.11, invoke 2.2.0.

Are new lines not supported or am I missing something?

**Comment by kuwv** [comment]
@joaonc Have you tried a multiline string?

```
"""
This is an example.
With a newline.
"""
```

Additionally, you can right-justify your syntax with `textwrap.dedent`:

```
from textwrap import dedent
@task(
    help={
        'my_option': dedent(
            """\
            Long description.
            Second line.
            """
        )
    },
)
def my_task(c, my_option: str):
    ...
```

I haven't tested this but I've used something similar.
